main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -287,8 +288,18 @@ class LanguageServer(port: Int) extends WebSocketServer(new InetSocketAddress("l
           // Compute elapsed time.
           val e = System.nanoTime() - t
 
-          // Send back a status message.
-          ("id" -> requestId) ~ ("status" -> "success") ~ ("time" -> e)
+          // Compute Code Quality hints.
+          val hints = CodeQuality.run(root)(flix)
```

Comment:
So this is only available via the LSP. I think it would be nice if this were accessible via command line as well.

Reply:
Perhaps under `--hints` or?

Reply:
Sure